kakarot_scripts/constants.py

```diff
@@ -182,6 +182,7 @@ class ChainId(IntEnum):
     16,
 )
 SOURCE_DIR = Path("src")
+SOURCE_PATH = Path("solidity_contracs/src")
```

Comment:
naming is confusing and this needs to be taken from the foundry file (see get_solidity_artifacts)